ste/mgr-JobMgr.go

```diff
@@ -961,7 +964,14 @@ func (jm *jobMgr) scheduleJobParts() {
 				go jm.poolSizer()
 				startedPoolSizer = true
 			}
-			jobPart.ScheduleTransfers(jm.Context())
+			getCredential := true
```

Comment:
also nit: should we call this getSourceCredential? I know in AzCopy they seem to be very explicit about source vs dest names